CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- `data.scalr_endpoint`: allow to obtain scalr_endpoint by name ([#101](https://github.com/Scalr/terraform-provider-scalr/pull/156))
+- `data.scalr_endpoint`: `id` become optional  ([#101](https://github.com/Scalr/terraform-provider-scalr/pull/156))
+- `data.scalr_endpoint`: added new optional attribute `name` ([#101](https://github.com/Scalr/terraform-provider-scalr/pull/156))
+- `data.scalr_endpoint`: added new optional attribute `env_id` ([#101](https://github.com/Scalr/terraform-provider-scalr/pull/156))
+- `data.scalr_webhook`: allow to obtain scalr_webhook by name ([#101](https://github.com/Scalr/terraform-provider-scalr/pull/156))
+- `data.scalr_webhook`: `id` become optional  ([#101](https://github.com/Scalr/terraform-provider-scalr/pull/156))
+- `data.scalr_webhook`: added new optional attribute `name` ([#101](https://github.com/Scalr/terraform-provider-scalr/pull/156))
+- `data.scalr_webhook`: added new optional attribute `env_id` ([#101](https://github.com/Scalr/terraform-provider-scalr/pull/156))
```

Comment:
Looks like this changelog could be squeezed to two list entries like

- `data.scalr_endpoint`: allow to obtain `scalr_endpoint` by name, added new optional attributes `name` and `env_id`, `id` become optional ([#156](https://github.com/Scalr/terraform-provider-scalr/pull/156))
- `data.scalr_webhook`: allow to obtain `scalr_webhook` by name, added new optional attributes `name` and `env_id`,  `id` become optional ([#156](https://github.com/Scalr/terraform-provider-scalr/pull/156))